src/components/raycaster.js

```diff
@@ -31,6 +31,15 @@ module.exports.Component = registerComponent('raycaster', {
     this.prevCheckTime = undefined;
     this.raycaster = new THREE.Raycaster();
     this.updateOriginDirection();
+    this.refreshObjects = this.refreshObjects.bind(this);
+  },
+
+  play: function () {
+    this.el.sceneEl.addEventListener('child-attached', this.refreshObjects);
```

Comment:
Rather than refresh all objects again, we can check if the child matches the selector, and append to array.